<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acFileMenu</class>
 <widget class="QMenu" name="acFileMenu">
  <property name="title">
   <string>&amp;File</string>
  </property>
  <action name="actionOpenData">
   <property name="icon">
    <iconset resource="../../Resources/Icons/icons.qrc">
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="icon">
    <iconset resource="../../Resources/Icons/icons.qrc">
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <addaction name="actionOpenData"/>
  <addaction name="actionSaveData"/>
  <addaction name="separator"/>
  <addaction name="actionExit"/>
 </widget>
 <resources>
  <include location="../../Resources/Icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
